pom.xml

```diff
@@ -44,14 +48,53 @@
 
   <build>
     <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <version>${dependency.plugin.version}</version>
+        <executions>
+          <execution>
+            <id>unpack-sdk-grammars</id>
+            <phase>generate-sources</phase>
+            <goals>
+              <goal>unpack</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <artifactItems>
+            <artifactItem>
+              <groupId>eu.europa.ted.eforms</groupId>
+              <artifactId>eforms-sdk</artifactId>
+              <version>${eforms.sdk.version}</version>
+              <type>jar</type>
+              <includes>eforms-sdk/efx-grammar/**/*.g4</includes>
```

Comment:
Maybe this could a variable aswell like "eforms.sdk.efx.folder", but it is ok as it is right now.